### PR TITLE
entity 5512

### PR DESCRIPTION
**Pull request by scottrumsby**
#### bcgov/entity#5512

- updated the UI messaging in analyze-results.vue to better fit the app without auto-analysis approvals

##

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
Please update the PR to something meaningful (eg, "updated UI messaging") to make it easier to look through commits when looking for what change broke something :)

File: client/src/components/new-request/analyze-results.vue [hunk 1]
```diff
@@ -135,9 +135,7 @@
                 <v-col cols="12" class="copy-normal pt-2 pb-4">
                   <v-row justify="center">
                     <v-col cols="auto">
-                      Name is available for a
-                      {{ entityText === 'BC Corporation' && location.text === 'BC' ? '' : location.text }}
-                      {{ ' ' + entityText }}
+                      Your name looks good to send to BC Registries Names Staff for a final decision.
```
Comment (severinbeauvais) [inline client/src/components/new-request/analyze-results.vue:138]: The design in the ticket says "This name...".

Did this get changed? If not, please update. Thx.

Comment (severinbeauvais) [reply client/src/components/new-request/analyze-results.vue:138]: (Personally I prefer "Your name..." but it isn't up to me.)

**Review by cameron-freshworks — APPROVED** [review]
Looks good. After you verify the verbiage vs the design you're g2g 👍

**Comment by scottrumsby** [comment]
Changes introduced by this PR have since been added to the app by other PRs.  No longer needed-closing now

**Pull request closed without merging**